code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -3,8 +3,10 @@ var/list/slime_colours = list("rainbow", "grey", "purple", "metal", "orange",
 	"gold", "green", "adamantine", "oil", "light pink", "bluespace",
 	"cerulean", "sepia", "black", "pyrite")
 
+var/regex/slime_name_regex = new("/\\w+ (baby|adult) slime \\(\\d+\\)/")
```

Comment:
I haven't used /static before, do I move the var to the slime datum and call it var/static/regex/slime_name_regex ?